storage/storage-mq/src/api/pipelineExecutionEventHandler.ts

```diff
@@ -8,17 +8,17 @@ export default class PipelineExecutionEventHandler {
   }
 
   async handleSuccess (pipelineExecutedEvent: PipelineExecutedEvent): Promise<void> {
-    await this.contentRepository.saveContent(pipelineExecutedEvent.pipelineId, {
+    await this.contentRepository.saveContent(pipelineExecutedEvent.pipelineId.toString(), {
       pipelineId: pipelineExecutedEvent.pipelineId,
-      timestamp: pipelineExecutedEvent.timestamp,
-      data: pipelineExecutedEvent.data,
-      id: undefined
+      timestamp: pipelineExecutedEvent.timestamp ?? new Date(),
```

Comment:
This is kind of a workaround because the messages from the integration tests contain the `timestamp` field, but the actual events from the Pipeline service do not: https://github.com/jvalue/open-data-service/blob/master/pipeline/src/pipeline-config/publisher/amqpExecutionResultPublisher.ts#L30.

This is just one instance of a broader issue/difficulty that exists in the codebase: The type definitions of the amqp messages are always duplicated at producer and consumer site. Which will lead to diverging type definitions if not all producers and consumers are updated simultaneously.

Reply:
I totally agree - that's one of the challenges with a distributed architecture. We won't want to have shared libs for that because microservices want to avoid that (we are not there to enjoy the benefits of that)... When we introduce consumer-driven contract tests this should be solved. Until now, we have our system tests to make sure that the main functionality works... Details like that are not tested, so we have to live with these little mismatches for now ...